CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contribution Guidelines
+
+This document aims to provide guidelines for merging changesets into the sparrow master branch.
```

Comment:
This document aims to provide guidelines for merging changesets into the Sparrow master branch.